backend/tracim_backend/lib/core/user.py

```diff
@@ -560,6 +560,12 @@ def _ldap_authenticate(
         # INFO - G.M - 2018-11-22 - LDAP Auth
         data = ldap_connector.authenticate(login, password)
         if not data:
+            if user.auth_type == AuthType.UNKNOWN:
```

Comment:
I suggest you put a comment with reference to the issue. Otherwise, a maintainer will not understand why this special case is here

Reply:
done